web/.env.development

```diff
@@ -1,4 +1,5 @@
 NODE_ENV=development
-VUE_APP_TITLE=FINTO-Ehdotusjärjstelmä (DEV)
+VUE_APP_TITLE=FINTO-Ehdotusjärjestelmä (DEV)
 VUE_APP_BASE_DOMAIN=http://localhost:8080
-VUE_APP_GITHUB_CLIENT_ID=b7aca67b03ae97580e12
```

Comment:
Palautan tämän tuohon alkuperäiseen kun tämä on ok, tai lähinnä muutan tuon alkuperäisen appin redirect urlit oikeiksi. Mutta tämä tämän haaran toimivuuden kannalta hyvä olla vielä näin